tests/addon/test_unit.py

```diff
@@ -2967,3 +2985,50 @@ def test_get_daily_review_data_no_reviews(mocker, anki_session_with_addon_data):
         result = get_daily_review_summaries_since_last_sync(last_sync)
 
         assert result == []
+
+
+def test_send_daily_review_summaries_with_data(mocker):
+    last_summary_sent_date = (datetime.now() - timedelta(days=1)).date()
+    mock_summary = MagicMock()
+
+    mock_config = mocker.patch("ankihub.main.review_data.config")
+    MockAnkiHubClient = mocker.patch("ankihub.main.review_data.AnkiHubClient")
+    mock_get_daily_review_summaries = mocker.patch(
+        "ankihub.main.review_data.get_daily_review_summaries_since_last_sync"
+    )
+
+    mock_get_daily_review_summaries.return_value = [mock_summary]
+    mock_anki_hub_client = MockAnkiHubClient.return_value
+
+    send_daily_review_summaries(last_summary_sent_date)
+
+    mock_get_daily_review_summaries.assert_called_once_with(last_summary_sent_date)
+    mock_anki_hub_client.send_daily_card_review_summaries.assert_called_once_with(
+        [mock_summary]
+    )
+    mock_config.save_last_summary_sent_date.assert_called_once()
+    assert (
+        mock_config.save_last_summary_sent_date.call_args[0][0] <= datetime.now().date()
```

Comment:
:pick: I think it's better to be more strict here with the assertion.
```suggestion
        mock_config.save_last_summary_sent_date.call_args[0][0] == datetime.now().date()
```